packages/css/README.md

```diff
@@ -0,0 +1,43 @@
+# CSS definitions of the web platform
+
+This package contains CSS property definitions scraped from the latest versions of web platform specifications in [webref](https://github.com/w3c/webref), with fixes applied to ensure ([almost](#guarantees)) all CSS value definitions can be parsed with [CSSTree](https://github.com/csstree/csstree).
+
+# API
+
+The async `listAll()` method resolves with an object where the keys are spec shortnames, and the values are the result of parsing the CSS files as JSON. Example:
```

Comment:
```suggestion
The async `listAll()` method resolves with an object where the keys are spec shortnames, and the values are the data for that spec. Example:
```

The fact that JSON is used internally in the package need not show up here I think.